SPECS/cloud-init/cloud-init.spec

```diff
@@ -109,6 +109,7 @@ echo -e 'line1\nline2\nline3\ncloud-init-ca-certs.crt\n' > "${conf_file}"
 %define test_pkgs pytest-metadata unittest2 mock attrs iniconfig httpretty netifaces
 
 pip3 install --upgrade %{test_pkgs}
```

Comment:
Do we need this installation step if we're installing from the test requirements file?

Reply:
Nope, looks like we need all of them but `httpretty`.
https://github.com/canonical/cloud-init/blob/main/test-requirements.txt